docs/Learning-Environment-Examples.md

```diff
@@ -224,3 +224,22 @@ If you would like to contribute environments, please see our
         * Goalie: 4 corresponding to forward, backward, sideways movement.
     * Visual Observations: None
 * Reset Parameters: None
+
+## Walker
+
+![Walker](images/walker.png)
+
+* Set-up: A humanoid agent with multiple degrees of freedom.
+* Goal: The agents must move its body toward the goal direction as quickly as possible without falling.
+* Agents: The environment contains 11 agent linked to a single brain.
```

Comment:
11 agent*s*

Why 11 agents?

Reply:
Somewhat arbitrarily choose. We want enough to speed up training, but not too many as to slow Unity/communication down. Also agents have to be spread apart, and if they are too far from the center of the scene, physics calculations become inaccurate.